utils/fmoconfig.py

```diff
@@ -75,11 +76,40 @@ def save_config(self) -> None:
         # Whenever fmo-config changed, rewrite ddp config
         self.write_vmddp_config()
```

Comment:
I notice the `write_vmpf_config()` is called in DPF functions,
Should `write_vmddp_config()` be re-written the same way? 
It is now inside `save_config()`, which write the ddp config every time there're changes in the config, regardless what the changes are.

Reply:
Yes, you should update your config only when it has changed

Reply:
Thanks, I'll mark this approve and make that change once this is merged